app/src/main/res/xml/preferences.xml

```diff
@@ -16,29 +16,5 @@
   -->
 
```

Comment:
It's necessary. Especially if we want to add any non dynamic preferences or preference groups such as "Use Imperial Units"